# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1324013159 28800
#      Thu Dec 15 21:25:59 2011 -0800
# Branch stable
# Node ID 324b2ec6214da8c0a4cb2a28a1c96864e5e0d1cd
# Parent  efba69175989a6831f668c1a8a81f72cf83befff
pt-eval.cc: Eliminate unused variable compile warning

* pt-eval.cc (visit_simple_for_command): Eliminate 'bool quit'
variable which is unused and causing compile warning.

diff --git a/src/pt-eval.cc b/src/pt-eval.cc
--- a/src/pt-eval.cc
+++ b/src/pt-eval.cc
@@ -318,17 +318,16 @@ tree_evaluator::visit_simple_for_command
 
     if (rhs.is_range ())
       {
         Range rng = rhs.range_value ();
 
         octave_idx_type steps = rng.nelem ();
         double b = rng.base ();
         double increment = rng.inc ();
-        bool quit = false;
 
         for (octave_idx_type i = 0; i < steps; i++)
           {
             // Use multiplication here rather than declaring a
             // temporary variable outside the loop and using
             //
             //   tmp_val += increment
             //
@@ -358,18 +357,16 @@ tree_evaluator::visit_simple_for_command
         quit_loop_now ();
       }
     else if (rhs.is_matrix_type () || rhs.is_cell () || rhs.is_string ()
              || rhs.is_map ())
       {
         // A matrix or cell is reshaped to 2 dimensions and iterated by
         // columns.
 
-        bool quit = false;
-
         dim_vector dv = rhs.dims ().redim (2);
 
         octave_idx_type nrows = dv(0), steps = dv(1);
 
         if (steps > 0)
           {
             octave_value arg = rhs;
             if (rhs.ndims () > 2)
